manifests/o/OctopusDeploy/Tentacle/6.1.942/OctopusDeploy.Tentacle.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: OctopusDeploy.Tentacle
+PackageVersion: 6.1.942
```

Comment:
The dev messed up?
![image](https://user-images.githubusercontent.com/15158490/131634425-39cf86d4-965b-45b7-8ded-8be62ec4fd07.png)

Reply:
Oof. I'll just close this then, check on the next build.

Reply:
An issue on the Octopus end has been raised with this to be addressed - [https://github.com/OctopusDeploy/Issues/issues/7298](https://github.com/OctopusDeploy/Issues/issues/7298).